botx/clients/methods/errors/no_permission.py

```diff
@@ -0,0 +1,51 @@
+"""Definition for "no permission" error."""
```

Comment:
`permissions` as name of module would be better, i think

Reply:
done